lin_actuator.c

```diff
@@ -21,10 +21,6 @@ bool lin_actuator_dac_init(void) {
   data_buf[1] = ((HIGH_DAC_VAL & 0x0f) << 4);
   i2c1_writeNBytes(HIGH_DAC_ADDRESS, data_buf, 2);
 
-  data_buf[0] = ((LOW_DAC_VAL & 0xf0) >> 4);
-  data_buf[1] = ((LOW_DAC_VAL & 0x0f) << 4);
-  i2c1_writeNBytes(LOW_DAC_ADDRESS, data_buf, 2);
-
```

Comment:
I would have sworn that I committed this yesterday. Wanna rebase onto origin/master?

Reply:
Ah, I don't think I pulled today.

Reply:
I didn't expect you to commit to master on my repo lol